kernel-scala/build.sbt

```diff
@@ -2,6 +2,11 @@
  * Copyright (c) Jupyter Development Team.
  * Distributed under the terms of the Modified BSD License.
  */
+
+import scala.util.Properties
+
+organization := "org.jupyter"
+
 name := "declarativewidgets"
 
 version := "0.1-SNAPSHOT"
```

Comment:
@lbustelo updated the PR.  publish jar depends on export release.  It uses a file as the intermediary between the two targets.  I was having an issue exporting as it was operating in a different shell.  Maybe we should treat the version setting differently to avoid this commit.